src/classnames.macro.js

```diff
@@ -35,13 +35,26 @@ export default createMacro(({ babel, references: { default: paths } }) => {
     return babel.types.arrayExpression(elements)
   }
 
+  function unpackObjectProperty(objectProperty) {
+    console.log(objectProperty)
```

Comment:
Is this stripped in final build? It is extremely verbose when testing anyway :)

Reply:
Uh no. It should not be there, development process leftover. Good catch.